include.js

```diff
@@ -5861,6 +5861,13 @@ let cards = [
     imageLink: './Art/Joy/triangle/trance.gif',
     author: 'Alex',
     githubLink: 'https://github.com/averde1973'
+  },
+  {
+    artName : "animationzone"
```

Comment:
Please fix syntax error (missing comma).

Reply:
i fixed it and update the repository,but i dont see any new commit in the repository
thank you.